neon_core/__init__.py

```diff
@@ -82,6 +87,16 @@ def setup_ovos_config():
 
 setup_ovos_config()
 
+# make holmesV Configuration.get() load neon.conf
+# TODO HolmesV does not yet support yaml configs, once it does
+#  Configuration.get() will be made to load the existing neon config files,
+#  for now it simply provides correct default values
+setup_ovos_core_config()
+
+neon_config_path = join(xdg.BaseDirectory.save_config_path("neon"),
+                        "neon.conf")
+write_mycroft_compatible_config(neon_config_path)
```

Comment:
this can overwrite an existing user config, should that be accounted for? even if we just add a log telling users to use yaml config

Reply:
Hmm; yeah, the idea is that 'user config' should be handled in yaml.. A log makes sense